staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset.go

```diff
@@ -232,7 +234,7 @@ const (
 // executing at each point where there is a change in that quantity,
 // because the metrics --- and only the metrics --- track that
 // quantity per FlowSchema.
-func (qs *queueSet) StartRequest(ctx context.Context, hashValue uint64, flowDistinguisher, fsName string, descr1, descr2 interface{}, queueNoteFn fq.QueueNoteFn) (fq.Request, bool) {
+func (qs *queueSet) StartRequest(ctx context.Context, width float64, hashValue uint64, flowDistinguisher, fsName string, descr1, descr2 interface{}, queueNoteFn fq.QueueNoteFn) (fq.Request, bool) {
```

Comment:
To simplify the changes (according to the rule the review is exponentially expensive to the size of PR), I suggest to first make a very simple PR that you will just:
- add the SetsInUse fields (and do all the plubming that you did here)
- as the first line in this function set;
```
width := 1
```

This would clearly split this PR into two parts and make the review easier imho.

Reply:
@wojtek-t I opened https://github.com/kubernetes/kubernetes/pull/102185